ibm/mas_devops/roles/mongodb/tasks/providers/community/install.yml

```diff
@@ -247,6 +267,48 @@
     - (mongodb_ce_version is version('0.7.8', '>=') and mongodb_cr.resources[0].status.version is defined) or mongodb_ce_version is version('0.7.0', '==')
     - (mongodb_ce_version is version('0.7.8', '>=') and mongodb_cr.resources[0].status.version == expected_mongodb_version) or mongodb_ce_version is version('0.7.0', '==')
 
+# 9.1 Create MongoDb service monitor
+# -----------------------------------------------------------------------------
+- name: "community : install : Check for existing service monitor"
+  kubernetes.core.k8s_info:
+    api_version: monitoring.coreos.com/v1
+    kind: ServiceMonitor
+    name: mas-mongo-ce-service-monitor
+    namespace: "{{ mongodb_namespace }}"
+  register: mongoce_servicemonitor_info
+  when:
+    - mongodb_ce_version is version('0.7.8', '>=')
+
+- name: "community : install : Create new MongoDb service monitor"
+  when:
+    - mongoce_servicemonitor_info.resources is defined
+    - mongoce_servicemonitor_info.resources | length == 0
+    - mongodb_ce_version is version('0.7.8', '>=')
+  kubernetes.core.k8s:
+    apply: yes
+    definition: "{{ lookup('template', 'templates/community/servicemonitor.yml.j2') }}"
+
+# 9.2 Create MongoDb Grafana dashboard
+# -----------------------------------------------------------------------------
+- name: "community : install : Check for existing Grafana dashboard"
+  kubernetes.core.k8s_info:
+    api_version: integreatly.org/v1alpha1
+    kind: GrafanaDashboard
+    name: mongodb-overview
+    namespace: "{{ mongodb_namespace }}"
+  register: mongoce_grafana_dashboard_info
+  when:
+    - mongodb_ce_version is version('0.7.8', '>=')
```

Comment:
Can we add a guard against this task like we do for kafka https://github.com/ibm-mas/ansible-devops/blob/3ebec8c56a69959001ee6388d28f5e376996cd52/ibm/mas_devops/roles/kafka/tasks/provider/redhat/install.yml#L178 that will check if the integreatly.org api is installed or not. This will allow the mongodb role to continue to run if cluster_monitoring was not run.

Note: you would need to add a lookup for the api like here https://github.com/ibm-mas/ansible-devops/blob/3ebec8c56a69959001ee6388d28f5e376996cd52/ibm/mas_devops/roles/kafka/tasks/provider/redhat/install.yml#L41

Reply:
@whitfiea requested changes have been made, currently building